tools/render-test/render-test-main.cpp

```diff
@@ -555,21 +555,30 @@ SLANG_TEST_TOOL_API SlangResult innerMain(Slang::StdWriters* stdWriters, SlangSe
 			return SLANG_FAIL;
 	}
 
+    StringBuilder rendererName;
+    rendererName << "[" << RendererUtil::toText(gOptions.rendererType) << "] ";
```

Comment:
Why this choice of punctuation - square braces for the renderer type and quotes for the adapter?

Also as a tiny nit, it might be nicer to only add the space after `]` if there is an adapter name, rather than unconditionally.